tests/routes/visitor_check.test.js

```diff
@@ -21,7 +21,7 @@ test('POST /api/visit/check | viable name & email & exists in database', t => {
     .post('/api/visit/check')
     .set('authorization', token)
     .send(successPayload)
-    .expect(200)
+    .expect(409)
```

Comment:
Is this test correctly named? It is meant to be a success case but returns a `409`.

Reply:
Where does it say it's meant to be a success case?